packages/pieces/sendy/project.json

```diff
@@ -0,0 +1,44 @@
+{
+  "name": "pieces-sendy",
+  "$schema": "../../../node_modules/nx/schemas/project-schema.json",
+  "sourceRoot": "packages/pieces/sendy/src",
+  "projectType": "library",
+  "targets": {
+    "build": {
+      "executor": "@nx/js:tsc",
+      "outputs": [
+        "{options.outputPath}"
+      ],
+      "options": {
+        "outputPath": "dist/packages/pieces/sendy",
+        "tsConfig": "packages/pieces/sendy/tsconfig.lib.json",
+        "packageJson": "packages/pieces/sendy/package.json",
+        "main": "packages/pieces/sendy/src/index.ts",
+        "assets": [
+          "packages/pieces/sendy/*.md"
+        ],
+        "buildableProjectDepsInPackageJsonType": "dependencies",
+        "updateBuildableProjectDepsInPackageJson": true
+      }
+    },
+    "publish": {
+      "command": "node tools/scripts/publish.mjs pieces-sendy {args.ver} {args.tag}",
+      "dependsOn": [
+        "build"
+      ]
+    },
+    "lint": {
+      "executor": "@nx/eslint:lint",
+      "outputs": [
+        "{options.outputFile}"
+      ],
+      "options": {
+        "lintFilePatterns": [
+          "packages/pieces/sendy/**/*.ts",
+          "packages/pieces/sendy/package.json"
```

Comment:
Can you please remove that to fix the lint issue.

I tried to fix it but didn't have permission to edit this PR

Reply:
Sure. Interesting that I did not add this. Maybe my tooling did?